common/test/acceptance/tests/studio/test_studio_settings_certificates.py

```diff
@@ -52,7 +52,7 @@ def create_and_verify_certificate(self, course_title_override, existing_certs, s
             certificate.signatories[idx].name = signatory['name']
             certificate.signatories[idx].title = signatory['title']
             certificate.signatories[idx].organization = signatory['organization']
-            certificate.signatories[idx].upload_signature_image('Signature-{}.png'.format(idx))
+            certificate.signatories[idx].upload_signature_image('Signature_{}.png'.format(idx))
```

Comment:
Actually, I keep the old file in repository so i can used them in my tests. I mean invalid .png or exceeding dimesnion etc. what you say ?